Traceback (most recent call last):
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\jpocampo\.conda\envs\nuevo2-env\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#df = pd.read_pickle(project_path / 'data' / _PICKLED_DATA_FILENAME)
df = pd.read_pickle(r"C:\Proyectos DigitalizaciÃ³n\Bancolombia\CB_informes_Ubi\Informe_semanal_v2\data\data_weekly_report.pkl")

# Legacy code (including the library) expects these column names
# but the new Ubidots library returns more specific column names
# so renaming is necessary. TODO: rework the Report library
# so that it uses these more descriptive column names.
df = df.rename(columns={'variable_label':'variable','device_label':'device',})

df = df.sort_values(by=['variable','datetime'])
df = pro.datetime_attributes(df)

# Agrego esta linea codigo para quitar estas dos sedes que generar ruido por datos altos
df = df[~df["device_name"].isin(["BC 829 - Unicentro Cali", "BC 454 - Quinta Avenida 2"])]

df_bl, df_st = pro.split_into_baseline_and_study(df, baseline=cfg.BASELINE, study=cfg.STUDY, inclusive='left')

study_daterange = pd.Series(pd.date_range(start=cfg.STUDY[0], end=cfg.STUDY[1], freq='D'))

------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mMemoryError[0m                               Traceback (most recent call last)
Cell [1;32mIn[4], line 2[0m
[0;32m      1[0m [38;5;66;03m#df = pd.read_pickle(project_path / 'data' / _PICKLED_DATA_FILENAME)[39;00m
[1;32m----> 2[0m df [38;5;241m=[39m [43mpd[49m[38;5;241;43m.[39;49m[43mread_pickle[49m[43m([49m[38;5;124;43mr[39;49m[38;5;124;43m"[39;49m[38;5;124;43mC:[39;49m[38;5;124;43m\[39;49m[38;5;124;43mProyectos DigitalizaciÃ³n[39;49m[38;5;124;43m\[39;49m[38;5;124;43mBancolombia[39;49m[38;5;124;43m\[39;49m[38;5;124;43mCB_informes_Ubi[39;49m[38;5;124;43m\[39;49m[38;5;124;43mInforme_semanal_v2[39;49m[38;5;124;43m\[39;49m[38;5;124;43mdata[39;49m[38;5;124;43m\[39;49m[38;5;124;43mdata_weekly_report.pkl[39;49m[38;5;124;43m"[39;49m[43m)[49m
[0;32m      4[0m [38;5;66;03m# Legacy code (including the library) expects these column names[39;00m
[0;32m      5[0m [38;5;66;03m# but the new Ubidots library returns more specific column names[39;00m
[0;32m      6[0m [38;5;66;03m# so renaming is necessary. TODO: rework the Report library[39;00m
[0;32m      7[0m [38;5;66;03m# so that it uses these more descriptive column names.[39;00m
[0;32m      8[0m df [38;5;241m=[39m df[38;5;241m.[39mrename(columns[38;5;241m=[39m{[38;5;124m'[39m[38;5;124mvariable_label[39m[38;5;124m'[39m:[38;5;124m'[39m[38;5;124mvariable[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mdevice_label[39m[38;5;124m'[39m:[38;5;124m'[39m[38;5;124mdevice[39m[38;5;124m'[39m,})

File [1;32m~\.conda\envs\nuevo2-env\lib\site-packages\pandas\io\pickle.py:208[0m, in [0;36mread_pickle[1;34m(filepath_or_buffer, compression, storage_options)[0m
[0;32m    205[0m     [38;5;28;01mwith[39;00m warnings[38;5;241m.[39mcatch_warnings(record[38;5;241m=[39m[38;5;28;01mTrue[39;00m):
[0;32m    206[0m         [38;5;66;03m# We want to silence any warnings about, e.g. moved modules.[39;00m
[0;32m    207[0m         warnings[38;5;241m.[39msimplefilter([38;5;124m"[39m[38;5;124mignore[39m[38;5;124m"[39m, [38;5;167;01mWarning[39;00m)
[1;32m--> 208[0m         [38;5;28;01mreturn[39;00m [43mpickle[49m[38;5;241;43m.[39;49m[43mload[49m[43m([49m[43mhandles[49m[38;5;241;43m.[39;49m[43mhandle[49m[43m)[49m
[0;32m    209[0m [38;5;28;01mexcept[39;00m excs_to_catch:
[0;32m    210[0m     [38;5;66;03m# e.g.[39;00m
[0;32m    211[0m     [38;5;66;03m#  "No module named 'pandas.core.sparse.series'"[39;00m
[0;32m    212[0m     [38;5;66;03m#  "Can't get attribute '__nat_unpickle' on <module 'pandas._libs.tslib"[39;00m
[0;32m    213[0m     [38;5;28;01mreturn[39;00m pc[38;5;241m.[39mload(handles[38;5;241m.[39mhandle, encoding[38;5;241m=[39m[38;5;28;01mNone[39;00m)

[1;31mMemoryError[0m: 
MemoryError: 

